spec/spec_helper.cr

```diff
@@ -175,6 +175,7 @@ def run(code, filename = nil, inject_primitives = true, debug = Crystal::Debug::
 
     compiler = Compiler.new
     compiler.debug = debug
+    compiler.flags.concat flags if flags
```

Comment:
Nice! This is compatible with my refactor in https://github.com/crystal-lang/crystal/pull/7282 :)